styles/Home.module.css

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Since this PR adds styled-components I think it'd be easiest for us to maintain CSS if we stuck to just one CSS-design, e.g. styled-components. We don't have to do this in this PR, but it'd be good to as a task to the project board to convert any .css files to styled components. What do you think?

Reply:
the Card style isn't final. I just wanted to show something before the meeting WED so the group can see how the Card will work. Ok maybe we should convert to styled components